Lab1_Operationalizing_Pytorch_with_Mlflow/src/bert-classifier/requirements.txt

```diff
@@ -1,10 +1,18 @@
-torch>=1.6.0
+torch==1.7.1+cpu
```

Comment:
I think we can tidy up the requirements and conda files now that we have this fixed. That's for another PR though.

Reply:
see #17

Reply:
Agreed.